### ISSUE TITLE
publicbody - law issues

### ISSUE THREAD

**Issue by fin** [stefanw/froide#145]
before i fix a pull request that might break things:

it seems like currently, PublicBody.laws is being ignored even if explicitly set; instead, the law used is selected using PublicBody.default_law, which returns the first non-meta law of PublicBody.jurisdiction

is this intended; should we retire PublicBody.laws?
i'd like to change PublicBody.default_law to

1. prioritize PublicBody.laws, if set
2. order laws (either PublicBody.laws or PublicBody.default_law) by priority, which also seems to be ignored.

how's that?

-f

**Comment by stefanw** [comment]
Makes sense. `default_law` was supposed to be the "getter" that makes a smart choice.
